src/tools/diagnostics/updatecenter_troubleshooter.py

```diff
@@ -0,0 +1,1337 @@
+#!/usr/bin/env python
```

Comment:
Consider README.md in the diagnostics folder explaining to say a customer / CSS how to use it. Overall:
- how to launch it
- how to capture the output